### PR TITLE
Bch main

### PR DESCRIPTION
**Pull request by blaisefonseca**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,7 +5,8 @@
     "@bower_components/ammap3": "amcharts/ammap3#3.21.6",
     "@bower_components/angular-flex": "scbd/angular-flex.git#2.2.0",
     "@bower_components/angular-localizer": "funkjunky/ngLocalizer.git",
-    "@bower_components/cbd-forums": "scbd/forums#818a1c4082d5db4ac0c5d4b9f7c422e898516e4a",
+    "@bower_components/angular-loggly-logger": "scbd/angular-loggly-logger#2c67428b49951e18bd03cbdd4834d939ba8fba74",
```
Comment (blaisefonseca) [inline package.json]:
```suggestion
```

File: app/components/scbd-angularjs-controls/form-control-directives/km-link.js [hunk 2]
```diff
@@ -75,340 +56,343 @@ app.directive('kmLink', function() {
...
+						$scope.editor.progress.style   = "error";
+				};
 
-								if (!$scope.extensions || !$scope.extensions.length)
-										$scope.editor.save();
-						};
+				//====================
+				//
+				//====================
+				$scope.safeApply = function(fn)
```
Comment (sbilodeau) [inline app/components/scbd-angularjs-controls/form-control-directives/km-link.js:381]: we should use `$applyAsync()` in the future.

File: app/css/bch/lmo-construct.css [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+        table.lmo-transcript.antisenseStop
+        {
+            direction:ltr;
+            font-family:Verdana;
+            font-size:10px!important;
+            line-height:14px;
+            text-align:center;
+            display:inline-block;
+            /* IE6 / IE7 hack */
+            *display:inline;
```
Comment (sbilodeau) [inline app/css/bch/lmo-construct.css:18]: probably we can delete

File: app/css/bch/lmo-construct.css [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+                
+        table.lmo-transcript.sense      td.left,
+        table.lmo-transcript.senseStop td.left
+        {
+            width:  25px;
+            height: 50px;
+            /*border-top:    25px solid #CBDDEF;
+            border-bottom: 25px solid #CBDDEF;
+            border-left:   25px solid transparent;*/			
+            background:url('https://bch.cbd.int/bch/images/LmoConstruct/right.svg') 5px 0px no-repeat;
```
Comment (sbilodeau) [inline app/css/bch/lmo-construct.css:100]: we will have to move them as resources

File: app/templates/bch.ejs [hunk 5]
```diff
@@ -4,8 +4,9 @@
 <head>
     <base href="<%=baseUrl%>">
     <meta charset="utf-8">
-    <title>Biosafety Clearing-House</title>
-    <meta name="description" content="The Biosafety Clearing-House">
+    <title ng-bind="ngMeta.title"></title>
+    <meta name="description" content="{{ngMeta.description}}">
+    <link name="canonical" rel="canonical" href="{{ngMeta.canonical}}" />
```
Comment (sbilodeau) [inline app/templates/bch.ejs:9]: this ng-meta is a bit stupid. the title/meta/link should be added dynamically using dom manipulation when value is set/cleared 
eg: (a bit more sophisticated)
```
function setTitle(title) {
  var head = $('head');
  if(title){ 
    head.find('title').text(title)
    head.find('meta[property="og:title"]').attr('content', title)
  }
  else
  {
    head.find('title').remove()
    head.find('meta[property="og:title"]').remove()
  } 
}

```

File: server.js [hunk 6]
```diff
@@ -26,39 +24,47 @@ if(!appVersion || appVersion.trim()==''){
 app.set('view engine', 'ejs');
 app.use(cookieParser());
 
+//special case for compression as prod files are compressed from cloud-front.
+//use express compress when compress varibale is true. 
+if(process.env.COMPRESS=='true'){
```
Comment (sbilodeau) [inline server.js:29]: is this for dev?
can we use this instead?
https://docs.traefik.io/middlewares/compress/

**Pull request merged**